jdaviz/configs/specviz2d/tests/test_parsers.py

```diff
@@ -18,11 +18,44 @@ def test_2d_parser(specviz2d_helper):
     assert PRIHDR_KEY not in dc_0.meta
     assert 'header' not in dc_0.meta
     assert dc_0.meta['DETECTOR'] == 'MIRIMAGE'
+    assert dc_0.get_component('flux').units == 'MJy / sr'

```

Comment:
This will fail until we get a new `glue-astronomy` release with the upstream fix.